### PR TITLE
feat: :sparkles: cca usergroup initial code

### PR DESCRIPTION
**Pull request by arunasd463**

### CODE REVIEW

File: src/pages/group/[groupName]/user/download-logs.tsx
Comment (prakhar-s) [inline src/pages/group/[groupName]/user/download-logs.tsx:1]: do we need user download logs in CCA?

File: yarn.lock
Comment (prakhar-s) [inline yarn.lock:1]: Just verify that we don't install unnecessary  dependencies and the versions don't cause conflicts.

**Review by prakhar-s — CHANGES_REQUESTED** [review]

File: src/components/@core/activity/activity-types.ts [hunk 1]
```diff
@@ -3,11 +3,14 @@ export default {
...
   DATA_UPDATED: "Data Updated",
   DATA_CREATED: "Data Created",
   DATA_COMMENTED: "Data comment",
   PERMISSION_REMOVED: "Permission removed",
   PERMISSION_ADDED: "Permission added",
-  FOLLOWER_ADDED: "Follower added",
-  FOLLOWER_REMOVED: "Follower removed"
+  POSTED_RESOURCE: "Posted resource",
+  REMOVED_RESORUCE: "Removed resoruce",
+  UNFEATURED: "UnFeatured"
```
[the remark this answers is not in the document]
Comment (arunasd463) [reply src/components/@core/activity/activity-types.ts:15]: Its used as a conditional check in ContentBox . we need it .

File: src/components/@core/table/index.tsx
[the remark this answers is not in the document]
Comment (arunasd463) [reply src/components/@core/table/index.tsx:1]: There was an extra property called getCheckboxProps , added it and resused existing property .

File: src/pages/group/[groupName]/user/download-logs.tsx
Comment (arunasd463) [reply src/pages/group/[groupName]/user/download-logs.tsx:1]: yes we have download logs in cca.

File: src/components/pages/group/edit/form.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    allowUserToJoin
+  } = groupInfo;
+
+  const hForm = useForm<any>({
+    mode: "onChange",
+    resolver: yupResolver(
+      Yup.object().shape({
+        name: Yup.string().required(),
+        speciesGroupId: Yup.array().required(),
+        habitatId: Yup.array().required(),
```
[the remark this answers is not in the document]
Comment (arunasd463) [reply src/components/pages/group/edit/form.tsx:51]: not able to remove this , backend requires this .

**Review by prakhar-s — APPROVED** [review]

**Pull request merged**